firmware/lucidgloves-firmware/src/Main.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <mutex>
+#include "Controller/Gesture.h"
+#include "Controller/Haptics.h"
+#include "Encoding/AlphaEncoding.h"
+#include "Encoding/LegacyEncoding.h"
+#include "Communication/SerialCommunication.h"
+#include "Communication/BTSerialCommunication.h"
```

Comment:
only the `Encoding/IEncoding.h` `Communication/ICommunication.h` need to be included in the header, the implementation includes can be moved to the cpp file, but fine as they are really.

Reply:
good call, will fix